chimney/src/main/scala/io/scalaland/chimney/ValueProvider.scala

```diff
@@ -18,12 +18,12 @@ object ValueProvider extends ValueProviderDerivation {
 
 trait ValueProviderDerivation {
 
-  implicit def hnilCase[FromG <: HList, From, FromT, ToT, L <: Symbol, Modifiers <: HNil]
+  implicit def hnilCase[Modifiers <: HNil, FromG <: HList, From, FromT, ToT, L <: Symbol]
```

Comment:
No strong reason, it should be equivalent, but I believe that it should speed up type inference a bit.